astrokat/observe_main.py

```diff
@@ -340,17 +351,16 @@ def run_observation(opts, kat):
         # Only observe targets in valid LST range
         [start_lst, end_lst] = get_lst(observation_cycle['LST'])
 
-
         # Verify that it is worth while continuing with the observation
         # The filter functions uses the current time as timestamps
         # and thus incorrectly set the simulation timestamp
         if not kat.array.dry_run:
             # Quit early if there are no sources to observe
-            if len(catalogue.filter(el_limit_deg=opts.horizon)) == 0:
+            if len(catalogue.filter(el_limit_deg=local_horizon)) == 0:
```

Comment:
We need the horizon value to be set in `opts.horizon`, since this is how it gets into session (when we call `session.standard_setup` on line 386).  Rather drop the `local_horizon` variable, and stick with `opts.horizon`.

Reply:
did the assignment in the wrong place -- corrected it to assign to opts.horizon to ensure sessions has the value and then only use the session value in the observation, as it should be
need to build a test to check this